### PR TITLE
Ft blog page

### PR DESCRIPTION
**Pull request by MugabeIgnace**

### CODE REVIEW

File: UI/pages/blog/blog.css [hunk 1]
```diff
@@ -6,7 +6,7 @@
     --clr-primary2: #102B3C;
     --clr-white: white;
     --clr-line: #999;
-    --clr-hover: #bdffff;
+    --clr-hover: #bdcfff6b;
```
Comment (MugabeIgnace) [inline UI/pages/blog/blog.css:9]: That is good for hovering

**Comment by MugabeIgnace** [comment]
>

File: UI/pages/blog/blog.css [hunk 2]
```diff
@@ -1,16 +1,15 @@
-/* Googlefont Poppins CDN Link */
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap');
 
 :root {
-    --clr-background: #102B3C;
     --clr-primary: #0C73B5;
+    --clr-secondary: #53ACE5;
+    --clr-primary2: #102B3C;
```
Comment (Kevin-ishimwe) [inline UI/pages/blog/blog.css:6]: great color changes

File: UI/pages/blog/blog.css [hunk 3]
```diff
@@ -27,251 +26,162 @@ hr {
...
 }
 
-body {
-    display: grid;
-    background-color: var(--clr-primary);
-    grid-template-areas:
-        "aside  main main"
-        "aside main main"
-    ;
+.dropdown {
```
Comment (Kevin-ishimwe) [inline UI/pages/blog/blog.css:95]: refactoring acknowledged

File: UI/pages/blog/blog.css [hunk 4]
```diff
@@ -27,251 +26,162 @@ hr {
...
-.left_hd {
-    display: flex;
-    background-image: url(../../../images/splash.avif);
-    margin: 0 1em;
-    border-radius: 0 0 20px 20px;
-    width: 100%;
-    height: 100%;
-    background-position: center;
-    background-size: cover;
+.dropdown_btn {
```
Comment (Kevin-ishimwe) [inline UI/pages/blog/blog.css:99]: design changes reviewed

**Pull request merged**